src/gui/shell.cpp

```diff
@@ -1237,11 +1244,15 @@ void Shell::keyPressEvent(QKeyEvent *ev)
 		this->setCursor(Qt::BlankCursor);
 	}
 
-	const QString inp{ Input::convertKey(*ev) };
-
+	QLocale* useLocale { &m_currentLocale };
+	const QString inp{ Input::convertKey(*ev, useLocale) };
```

Comment:
Consume directly, no need for `useLocale`:
`const QString inp{ Input::convertKey(*ev, &m_currentLocale) };`